apps/frontend/src/pages/app.vue

```diff
@@ -834,13 +813,9 @@ useSeoMeta({
             Mac
           </div>
           <div class="description apple">
-            <a :href="macLinks.appleSilicon" download="">
-              <DownloadIcon />
-              <span> Download for Apple Silicon </span>
-            </a>
-            <a :href="macLinks.intel" download="">
+            <a :href="macLinks.universal" download="">
               <DownloadIcon />
-              <span> Download for Intel </span>
+              <span> Download (Intel & Apple) </span>
```

Comment:
```suggestion
              <span> Download the beta </span>
```

No real reason to complicate things - be consistent with the Windows download link

Reply:
I received feedback from @Prospector suggesting that explicitly mentioning both CPU architectures helps clarifying to end users that both older and newer Macs are supported. What do you two think? Should we aim for extra consistency here?

Reply:
Applied!